docs/source/migration-guides.rst

```diff
@@ -0,0 +1,258 @@
+Migration guides
+================
+
+Migration from Waffle 2.2.0 to Waffle 2.3.0
+-------------------------------------------
+
+**Created monorepo**
+
+Packages :code:`waffle-cli`, :code:`waffle-chai`, :code:`waffle-provider`, :code:`waffle-compiler` were migrated to monorepo.
```

Comment:
This is not helpful for outside users. We should say this differently:
```
Waffle was internally migrated to a monorepo. Because of this you can now use parts of waffle individually. We provide the following packages:
- :code:`ethereum-waffle` - core package exporting everything
- :code:`ethereum-compiler` - compile your contracts programmatically
- :code:`ethereum-chai` - chai matchers for better unit testing
- :code:`ethereum-provider` - mock provider to interact with an in-memory blockchain
```